content/docs/concepts/staking/_index.md

```diff
@@ -55,37 +59,59 @@ For example, if a validator had suffered a slashing penalty equivalent to 1% of
+  
+Stake redemption takes place at the end of the epoch in which the unbonding period falls. At this point the validator's stake pool is reduced by the unbonded amount and Newton is returned to the staker.
 
-{{% alert title="Note" %}}It's important to note that Liquid Newton is validator-specific and as such is not 1:1 fungible with Liquid Newton bonded to a different validator. A validator may or may not have had slashing penalties applied and the redemption value of Liquid Newton may vary across validators according to their history.{{% /alert %}}
+{{% alert title="Note" %}}It's important to note that Liquid Newton is validator-specific and as such is not 1:1 fungible with Liquid Newton bonded to a different validator. A validator may or may not have had [slashing penalties](/glossary/#slashing-penalty) applied and the redemption value of Liquid Newton may vary across validators according to their fault slashing history.{{% /alert %}}
 
 ### Transferring Liquid Newton
 
 For details on how to transfer Liquid Newton from a stake delegation to another account, see:
 
 - How to [Transfer Liquid Newton](/delegators/transfer-lntn/).
 
-## Staking rewards
+## Penalty-Absorbing Stake (PAS)
 
-Staking rewards are a distribution of fee revenue entitlement to stakeholders delegating stake to the consensus committee. Reward distribution takes place _pro-rata_ to the share of total stake bonded to validators in the consensus committee. The fee revenue comes from the _base fees_ charged for computing transactions included in blocks committed to the system ledger. The optional _priority fee_ of a transaction is not included in the rewards pool but awarded to the block proposer according to the EIP 1559 transaction fee mechanism.
+Autonity implements a [_penalty absorbing stake (PAS)_](/glossary/#penalty-absorbing-stake-pas) model where validator [self-bonded](/glossary/#self-bonded) stake is slashed in priority to [delegated](/glossary/#delegated) stake when applying [slashing penalties](/glossary/#slashing-penalty) for accountability events.
 
-Staking rewards are collected by protocol and accumulate in a protocol account as blocks are processed throughout a block epoch. The fees are out of circulation until epoch end, at which point the protocol distributes them _pro rata_ to their share of the stake bonding the committee to the validator committee members. After this distribution, the rewards become _claimable_ by stake delegators. Delegators then claim fees in a "pull" model, at the frequency they choose. 
+Slashing priority is simply:
 
-For more detail on EIP 1559 and the distinction between _base fee_ and _priority fee_, see [Transaction fees](/concepts/system-model/#transaction-fees) in the System model.
+- [self-bonded](/glossary/#self-bonded) stake is slashed as first priority until exhausted
```

Comment:
Maybe we should mention that `selfUnbondingStake` has priority over `SelfBonded`.

Instead slashing of delegated stake is divided pro rata between the unbonding one and the "non-unbonding" one.

Reply:
OK, I've edited this paragraph and bullet points to make this clear:

>Slashing priority is simply:
>
>- [self-bonded](/glossary/#self-bonded) stake is slashed as first priority until exhausted. If the validator has unbonding stake, then the unbonding stake is slashed first before bonded stake.
>- [delegated](/glossary/#delegated) stake is slashed as second priority when the slashing amount exceeds the amount of self-bonded stake available. If the delegator has unbonding stake, then the unbonding stake and bonded stake are slashed _pro rata_ with equal priority.

I think that states it as simply as possible.

Reply:
See commit 2401d40bc56baeb2145b5857c022b9a41d88eefb